src/index.ts

```diff
@@ -0,0 +1,11 @@
+import app from "./app";
+
+import dotenv from "dotenv";
+
+dotenv.config();
+
+const PORT = process.env.PORT || 5000;
+
+app.listen(PORT, () => {
```

Comment:
Connect main app with sequelize

Reply:
I will need some time to figure out sequilize and sequilize through typescript :)

Reply:
Made my solution with analyzing Vladimir's code,  
hope that in the future I will be much more original :)

Reply:
Sure. Take some time and check documentation.